src/main/java/Deadline.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Implements constructor methods for Deadline.
+ */
+public class Deadline extends Task {
+    protected Date deadlineDate;
+    protected Timing deadlineTime;
+
+    /**
+     * Instantiates Deadline object.
+     * @param description Description of deadline command.
+     * @param deadlineDate Date of deadline.
+     */
```

Comment:
As you have multiple Deadline constructors, I think it would be great if you commented on what each of them did differently and how the extra parameters influence how the object is instantiated.